nntrainer/layers/nnstreamer_layer.cpp

```diff
@@ -13,16 +13,52 @@
  * @todo: support transposing the data to support NCHW nntrainer data to NHWC
  * nnstreamer data
  */
+#include <fstream>
 
-#include <lazy_tensor.h>
+#include <common_properties.h>
 #include <nnstreamer_layer.h>
 #include <nntrainer_error.h>
 #include <nntrainer_log.h>
+#include <node_exporter.h>
 #include <parse_util.h>
 #include <util_func.h>
-
 namespace nntrainer {
 
+/**
+ * @brief NNSModelPath property
+ *
+ */
+class PropsNNSModelPath : public Property<std::string> {
+public:
+  static constexpr const char *key = "model_path"; /**< unique key to access */
+  using prop_tag = str_prop_tag;                   /**< property type */
+
+  /**
+   * @brief check is valid
+   *
+   * @param v value to check
+   * @return bool true if valid
+   */
+  bool isValid(const std::string &v) const override;
+};
+bool PropsNNSModelPath::isValid(const std::string &v) const {
```

Comment:
as validator must be different from tflite path or other files, unfortunately this can't be shared. (same for the tflite layer's model path)

Reply:
If you want to reduce code redundancy, PropsTflModelPath and PropsNNSModelPath can inherit a ModelPath which simply performs a file check and gives the key tag.

Reply:
NNSModelPath might not be file (eg) nnfw). For the flexibility, I will leave it as it is